apps/settings/js/modules/panel_cache.js

```diff
@@ -23,6 +23,7 @@ define(function(require) {
       LazyLoader.load(['shared/style/action_menu.css',
                        'shared/style/confirm.css',
                        'shared/style/progress_activity.css',
+                       '/shared/elements/gaia-icons/bidi-helper.css',
```

Comment:
Actually, according to this part of the README (and my tests), if I want it to be lazy loaded, I need to add it in both place.
```md
### Where to put panel styles?

Put settings core and root panel style in `settings.css`.
Put form-factor (phone or tablet..) specific styles in `settings_{phone}.css`.
If there's not much styles for your panel, put it in `app.css`, which will lazy loaded automatically.
If you create a separate css, you need include it in `index.html`, `panel_cache.js` to make it lazy loaded properly.
```
Did I misunderstand something?